sw/device/tests/BUILD

```diff
@@ -1062,3 +1062,18 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
```

Comment:
please update this file to fix build error
    deps = [
        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
        "//sw/device/lib/base:memory",
        "//sw/device/lib/base:mmio",
        "//sw/device/lib/dif:sensor_ctrl",
        "//sw/device/lib/runtime:log",
        "//sw/device/lib/testing:aon_timer_testutils",
        "//sw/device/lib/testing:clkmgr_testutils",
        "//sw/device/lib/testing:pwrmgr_testutils",
        "//sw/device/lib/testing/test_framework:check",
        "//sw/device/lib/testing/test_framework:ottf_main",
    ],

Reply:
thanks @jdonjdon! do you want me to merge the test of this PR? or do you want to do it separately?
Both are okay with me.

Reply:
please go ahead merge.
I don't have merge permit yet ;)